recipes/pytray/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytray" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytray-{{ version }}.tar.gz
+  sha256: e05cbf0ee750bdbff3a772fdd84d6bbcfd27a949127d0272cd0b691f968b359f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - async_generator
+    - deprecation
+    - python >=3.5
+
+test:
+  imports:
+    - pytray
+
+about:
+  home: https://github.com/muhrin/pytray.git
+  summary: A python tools library for baking pies
```

Comment:
Can we have a description that actually says what this package does?

Reply:
Done.